### PR TITLE
Move padding in dropdown menu from item*s* to item

### PR DESCRIPTION
**Pull request by philipperolet**
Required for implementation of create/manage assistant buttons design in selection menu, and search bar (issues #1622 and #1721), to have full-width border separators as per picture
image

Moreover seems rational that while Drowpdown.Item elements should have a standardized padding, custom stuff in the menu (non-itemized) may manage its padding

Each dropdown in code using DropdownMenu.Item(s) has been checked for the change to be non-breaking

### CODE REVIEW

**Comment by philipperolet** [comment]
Note: as it is, an alpha version of sparkle is used. Upon acceptance, I will merge, then in a separate PR immediately bump the sparkle version with updated package-lock.json

File: front/components/assistant/conversation/ConversationTitle.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export function ConversationTitle({
                 />
               </DropdownMenu.Button>
               <DropdownMenu.Items width={280}>
-                <div className="flex flex-col gap-y-4 py-4">
+                <div className="flex flex-col gap-y-4 px-4 py-4">
```
Comment (PopDaph) [inline front/components/assistant/conversation/ConversationTitle.tsx:51]: NIT: can simplify with `<div className="flex flex-col gap-y-4 p-4">` here and below, which is equivalent to what you wrote. 🙏🏻

File: front/components/assistant/conversation/AgentMessage.tsx [hunk 2]
```diff
@@ -307,7 +307,7 @@ function ErrorMessage({
           </DropdownMenu.Button>
           <div className="relative bottom-6 z-30">
             <DropdownMenu.Items origin="topLeft" width={320}>
-              <div className="flex flex-col gap-3 pb-3 pt-5">
+              <div className="flex flex-col gap-3 px-4 pb-3 pt-5">
```
Comment (PopDaph) [inline front/components/assistant/conversation/AgentMessage.tsx:310]: NIT: Is it on purpose to have top 5 bottom 3 and side 4? Could we simplify with 4 for all?

**Review by PopDaph — DISMISSED** [review]

File: sparkle/src/components/DropdownMenu.tsx [hunk 3]
```diff
@@ -300,7 +301,7 @@ DropdownMenu.Items = function ({
       <Menu.Items
         className={`s-absolute s-z-10 ${getOriginClass(
           origin
-        )} s-rounded-xl s-border s-border-structure-100 s-bg-structure-0 s-px-4 s-shadow-lg focus:s-outline-none dark:s-border-structure-100-dark dark:s-bg-structure-0-dark`}
+        )} s-rounded-xl s-border s-border-structure-100 s-bg-structure-0 s-shadow-lg focus:s-outline-none dark:s-border-structure-100-dark dark:s-bg-structure-0-dark`}
```
Comment (Duncid) [inline sparkle/src/components/DropdownMenu.tsx:304]: Removing the px will have effects!
To be watched when diployed.

**Review by Duncid — DISMISSED** [review]

File: front/components/assistant/conversation/AgentMessage.tsx (hunk 2 above)
Comment (philipperolet) [reply front/components/assistant/conversation/AgentMessage.tsx:310]: Figma is king

File: sparkle/src/components/DropdownMenu.tsx (hunk 3 above)
Comment (philipperolet) [reply sparkle/src/components/DropdownMenu.tsx:304]: I did check locally; but indeed

**Pull request merged**